py/selenium/webdriver/phantomjs/webdriver.py

```diff
@@ -61,6 +61,15 @@ def __init__(self, executable_path="phantomjs",
             raise
 
         self._is_remote = False
+    """ 
+    In order for the python with statement to work __enter__ and __exit__ methods must be deffined in the object.
```

Comment:
i don't think this comment is needed.  if anything, I would just state "__enter__ and __exit__ methods enable using this class as a context manager"